### ISSUE TITLE
no indication that response has finished for screen reader users if chat response takes more than 4 seconds

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#195813]
cc @jooyoungseo

We have a content media policy that prevents media playing if it's not occurred within 4 seconds of a corresponding user's gesture. 

We should alert in this case so that it's clear the response has come in.

**Comment by meganrogge** [comment]
@jooyoungseo, it occurred to me that we do indicate a response has come in for this case - by reading the content of the response. If that's not happening for you, it means the request failed for some reason.

**Comment by jooyoungseo** [comment]
Okay, I now understand. Do we want to close this issue now then?

**Comment by meganrogge** [comment]
Yes, I think we should because otherwise I think it would be redundant?
